plugin.video.youtube/resources/lib/youtube_plugin/kodion/utils/function_cache.py

```diff
@@ -106,16 +105,4 @@ def _seconds_difference(_first, _last):
         return cached_data
 
     def _optimize_item_count(self):
-        clear = False
```

Comment:
probably you don't need the method

Reply:
It's to override the method from the Storage class. Using database size for function cache and don't want to use item count as well. I'll add a comment to this method while I'm doing the fix up.